net/netatalk4/Portfile

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           meson 1.0
+PortGroup           openssl 1.0
+
+name                netatalk4
+conflicts           netatalk
+
+github.setup        Netatalk netatalk netatalk-4-1-2
+github.tarball_from archive
+revision            0
+
+checksums           rmd160  b6953e69144d6542c1b426015635d58f695cdba9 \
+                    sha256  012f2a32c4904f4010c1b924e5c01cf16dfc628cb32965be54479721f30aa30d \
+                    size    1327961
+
+categories          net
+license             GPL-2+
+maintainers         {openmaintainer github:trodemaster}
+
+description         Netatalk is a freely-available Open Source AFP fileserver.
+long_description    Netatalk is a freely-available Open Source AFP fileserver. \
+                    It allows Unix-like operating systems to serve as file \
+                    servers for Macintosh computers.
+
+homepage            https://netatalk.io
+
+depends_build       port:autoconf \
+                    port:automake \
+                    port:libtool
+
+depends_lib         port:db48 \
+                    port:libevent \
+                    port:libgcrypt \
+                    port:mysql8 \
+                    port:avahi
```

Comment:
On macOS, Netatalk will use native mDNS out of the box, so you don't need Avahi.

Reply:
Removed avahi, seems like libevent, libbcrypt & mysql8 are still needed.